CMakeLists.txt

```diff
@@ -192,6 +192,14 @@ function(mbed_generate_bin_hex target)
             "executable:"
         VERBATIM
     )
+
+    if(TARGET mbed-post-build-bin)
+        add_custom_target(mbed-post-build
```

Comment:
Why do need to create this custom target if a particular target(Musca b1) is adding their own custom target here https://github.com/ARMmbed/mbed-os/pull/14235/commits/75f8e6acafea602781511e1dee3bbb8341a638a1#diff-f3f63c03d654aaf1df8951f5770b77e265a473b470c33f0d9bea4ba5b9aa7e92R8 ?

Reply:
The reason for this custom target has been explained in the PR description.